### DISCUSSION TITLE
Controller models

### DISCUSSION THREAD

**Discussion by mamefan** [Raicuparta/nomai-vr#397] [Q&A] [answered]
Thanks for the mod. Sometimes, I see my Index controller models in my hands while playing. I never want to see them, so how can I keep that from happening?

**Comment by JpTiger** [comment]
I always figured that was a design choice. I'm on a Quest and the controllers tend to show up in menus. I don't remember seeing them in-game unless I was interacting with something menu-like (shipboard computer and maybe once while piloting the ship)

**Comment by artumino** [comment, marked as answer]
I should have solved the issue in #398 but it's not in a release right now.
As a workaround you can open and close the pause menu to make them disappear.

**Reply by artumino** [reply]
As of 1.1.4 controllers should stay hidden when a timeloop begins again

**Reply by mamefan** [reply]
Thank you!
